tests/unit/commands/validate/lib/openapi/notopenapi.yaml

```diff
@@ -0,0 +1 @@
+this is not yaml
```

Comment:
This looks unused. Was the suppose to be for a test?

Reply:
Yes it was for a test case that is no longer needed so have deleted.